### PR TITLE
fix: match copyright line in comments only

### LINKED ISSUE #32: License header checker should only look in headers
Example: https://github.com/googleapis/gcs-resumable-upload/pull/256

This file has a string that matches the copyright header regex and is resulting in an incorrect failure.

### PR DESCRIPTION
**Pull request by chingor13**
Fixes #32

### CODE REVIEW

File: packages/header-checker-lint/__snapshots__/header-checker-lint.js [hunk 1]
```diff
@@ -1,19 +1,3 @@
-/**
```
Comment (chingor13) [inline packages/header-checker-lint/__snapshots__/header-checker-lint.js]: Looks like snap-shot-it strips this out.

Comment (bcoe) [reply packages/header-checker-lint/__snapshots__/header-checker-lint.js]: Yeah, I'd noticed this, any thoughts? is there a heuristic way we could skip `__snapshot__` files, while still catching files that should have headers; it's been helpful to me catching a few legacy files that were simply missing headers.

Comment (chingor13) [reply packages/header-checker-lint/__snapshots__/header-checker-lint.js]: I'm looking into the configuration files in #34

**Comment by bcoe** [comment]
@chingor13 mind resolving the conflicts when you have a moment?

**Pull request merged**